lib/command.ts

```diff
@@ -332,3 +331,26 @@ Command.setReplyTransformer('hgetall', function (result) {
   }
   return result
 })
+
+class MixedBuffers {
+  length = 0
+  items = []
+
+  public push(x: string | Buffer) {
+    this.length += Buffer.byteLength(x);
+    this.items.push(x)
+  }
+
+  public toBuffer(): Buffer {
+    const result = Buffer.alloc(this.length);
+    let offset = 0;
+    this.items.forEach((b: Buffer|string) => {
```

Comment:
1 less closure would be created, not sure its going to be faster - but worth a try I believe

```js
for (const item of this.items) {

}
```

Reply:
done